plugins/routed-eni/driver/driver.go

```diff
@@ -181,7 +180,7 @@ func setupNS(hostVethName string, contVethName string, netnsPath string, addr *n
 	// Clean up if hostVeth exists.
 	if oldHostVeth, err := netLink.LinkByName(hostVethName); err == nil {
 		if err = netLink.LinkDel(oldHostVeth); err != nil {
-			return errors.Wrapf(err, "setup NS network: failed to delete old hostVeth", hostVethName)
+			return errors.Wrapf(err, "setup NS network: failed to delete old hostVeth %q", hostVethName)
```

Comment:
Ah, thanks for fixing this! I noticed the compile issues with 1.11, but didn't have time to find the cause of the problem yet. Had planned to look at it this week.